bign8/games
# Small Docker Images

Use board SVG's to shrink deployed asset.

## Verification Steps
* CI passes
* `make build`
* `make docker`
* `open localhsot:4000`
* Verify small docker image
  ```
  $ docker images | head
  REPOSITORY     TAG    IMAGE ID            CREATED             SIZE
  bign8/games    latest 8d075ad52b58        45 seconds ago      6.09 MB
  ```

## Future Ideas
* Have the home screen be an "attract" mode, showing the games playing themselves
  * Will help differentiate between checkers and chess
  * Common game practice
